src/ast/diagnostic.h

```diff
@@ -93,6 +93,12 @@ class Diagnostics {
     }
   }
 
+  // ok is the recommended short-hand for `has(Severity::Error)`.
+  bool ok() const
```

Comment:
hmmm, nitpicky but I worry about this now getting confused with the `OK` in result.

Reply:
Did you want me to capitalize it? It still means the same thing. You will just have:

```
return OK();
```

And:

```
ast.diagnostics().ok()
```

But you'll also have lots of:

```
auto ok = doThing();
if (!ok) {
   ...
 }
 ```
 
 Conceptually it's all the same, and has the same bool-like semantics (OK == good!). Where is the confusion coming in?

Reply:
Nevermind me, I think I was jet-lagged when I made this comment. `ast.diagnostics().ok()` is fine.